prime-angular-frontend/src/app/modules/site-registration/pages/remote-user-page/remote-user-page.component.ts

```diff
@@ -71,14 +72,22 @@ export class RemoteUserPageComponent extends AbstractEnrolmentPage implements On
     this.remoteUserIndex = route.snapshot.params.index;
   }
 
+  /**
+   * @description
+   * Remote user certifications specific to the local form.
+   */
+  public get remoteUserCertifications(): FormArray {
+    return this.form.get('remoteUserCertifications') as FormArray;
+  }
+
   public get selectedCollegeCodes(): number[] {
-    return this.formState.remoteUserCertifications.value
+    return this.remoteUserCertifications.value
       .map((certification: RemoteUserCertification) => +certification.collegeCode);
```

Comment:
Just curious, any particular reason to use unary operator + here to convert to `number` since `collegeCode` is already `number` type? Maybe should guard `collegeCode` field as `readonly` in `RemoteUserCertification` since it's immutable?